docs/manifest.json

```diff
@@ -219,6 +219,18 @@
                 ]
               ]
             },
+            {
+              "title": "Waitlist Components",
+              "items": [
+                [
+                  {
+                    "title": "`<Waitlist />`",
+                    "wrap": false,
+                    "href": "/docs/components/waitlist/overview"
+                  }
+                ]
+              ]
+            },
```

Comment:
what do you think about putting this under the "Authentication components" aka `/components/authentication` ?

Reply:
will there ever be other sign-up mode components? maybe we can rename the section to "Sign-up mode components"

If not, then I suppose we should change to "Waitlist component". Not plural component**s**, because there's only one component.